File: library/src/androidTest/java/com/raizlabs/android/dbflow/test/sql/MigrationModel.java
package com.raizlabs.android.dbflow.test.sql;

import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.structure.TestModel1;

/**
 * Author: andrewgrosner
 * Contributors: { }
 * Description:
 */
@Table(databaseName = MigrationDatabase.NAME)
public class MigrationModel extends TestModel1 {
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/sql/MigrationModel.kt
package com.raizlabs.android.dbflow.sql

import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.TestModel1

/**
 * Description:
 */
@Table(database = MigrationDatabase::class)
class MigrationModel : TestModel1()
